lib/commons/builder/membership_data.rb

```diff
@@ -11,24 +11,16 @@ def initialize(membership_rows, language_map, political_entity_kind)
   end
 
   def persons
-    membership_rows.map do |membership|
-      {
-        name: membership.name_object('name', language_map),
-        id: membership[:item].value,
-        identifiers: [
-          {
-            scheme: 'wikidata',
-            identifier: membership[:item].value,
-          },
-        ],
-        links: [
-          {
-            note: 'facebook',
-            url: membership[:facebook]&.value&.prepend('https://www.facebook.com/'),
-          },
-        ].select { |o| o[:url] },
-      }
-    end.uniq.sort_by { |p| p[:id] }
+    persons = {}
+    membership_rows.each do |membership|
+      unless persons[membership[:item].value]
+        persons[membership[:item].value] = person(membership)
+      end
+      if link = link(membership)
```

Comment:
This is completely correct, but just because this has the _shape_ of a common bug (equality / assignment mixup) I'd put the `link = link(membership)` on a line on its own, I think.

Generally, I found this section a little hard to read, which I think might be easier if the `membership[:item].value` was extracted to a `person_id` variable.